vmc/resource_vmc_cluster.go

```diff
@@ -215,6 +215,9 @@ func resourceClusterCreate(d *schema.ResourceData, m interface{}) error {
 			}
 			d.SetId(clusterID)
 		}
+		if *task.Status == "FAILED" {
```

Comment:
I think the error messages could be reworded. Something like... "Task failed to create instance" & "Task completed without creating instance"

Reply:
Sure.

Reply:
these errors are displayed in TF console. Is it required to explicitly state task in the message  to a TF user?